recipes-bsp/bootfiles/rpi-config/0001-config.txt-reintroduce-start_x.patch

```diff
@@ -0,0 +1,51 @@
+From ce27f7e22b2cd7453a425e08780a338a71301961 Mon Sep 17 00:00:00 2001
+From: Leon Anavi <leon.anavi@konsulko.com>
+Date: Mon, 20 Nov 2023 15:19:15 +0200
+Subject: [PATCH] config.txt: reintroduce start_x
+
+Reintroduce configuration "start_x". Based on the experience with
+Yocto/OpenEmbedded layer meta-raspberrypi, it has been observed
+that Raspberry Pi 4B 4GB may fail to enable the camera if
+"start_x=1" is at the end of the file. Therefore, "start_x=1"
+is expected in config.txt template and it has been set to replace
+the original occurrence, which is at the middle of the file.
+Also update revision and date stamp.
+
+Signed-off-by: Leon Anavi <leon.anavi@konsulko.com>
```

Comment:
Let's add the appropriate upstream status.

Reply:
Done. Thanks for the recommendation.